mongoengine/queryset/queryset.py

```diff
@@ -358,24 +355,39 @@ def insert(self, doc_or_docs, load_bulk=True, write_concern=None):
             return_one = True
             docs = [docs]
 
-        raw = []
         for doc in docs:
             if not isinstance(doc, self._document):
                 msg = ("Some documents inserted aren't instances of %s"
                        % str(self._document))
                 raise OperationError(msg)
             if doc.pk and doc._created:
-                msg = "Some documents have ObjectIds use doc.update() instead"
+                msg = 'Some documents have ObjectIds use doc.update() instead'
```

Comment:
Should be separate sentence / have a comma

Reply:
Done in:
* Upstream: https://github.com/MongoEngine/mongoengine/commit/996618a495ce0ccf2f0b81b0666e68c5af7fb71c
* Our fork: https://github.com/closeio/mongoengine/commit/7c06c31bba6c8d68ab92b6d8916bd1702457e76c